app/controllers/wish_kindergartens_controller.rb

```diff
@@ -0,0 +1,35 @@
+class WishKindergartensController < ApplicationController
+  def index
+    @kindergartens = Kindergarten.all
+    @wish = Wish.find(params[:wish_id])
+    @wish_kindergarten = WishKindergarten.new
+    @wish_kindergartens = @wish.wish_kindergartens
+  end
+
+  def create
+    @kindergartens = Kindergarten.order(:name)
+    @wish = Wish.find(params[:wish_id])
+    @wish_kindergarten = WishKindergarten.new(wish_kindergarten_params)
+    @wish_kindergarten.wish = @wish
+    @wish_kindergartens = @wish.wish_kindergartens
+
+    if @wish_kindergarten.save
+      redirect_to wish_wish_kindergartens_path(@wish), notice: "Uspešno ste dodali vrtić."
+    else
+      redirect_to wish_wish_kindergartens_path(@wish), notice: "Molimo Vas da unesete željeni vrtić."
```

Comment:
use `@wish_kindergarten.errors.full_messages.to_sentence` instead of hard coding error message